[pipelining] Retire PIPPY_VERBOSITY in favor of TORCH_LOGS=pp

kwen2501 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #126828

https://github.com/pytorch/pytorch/pull/126499/ established: 

`TORCH_LOGS=pp` --> info
`TORCH_LOGS=-pp` --> warn
`TORCH_LOGS=+pp` --> debug

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k

kwen2501 committed:
[pipelining] Retire PIPPY_VERBOSITY in favor of TORCH_LOGS=pp

[ghstack-poisoned]

wconstab reviewed (approved):
thank you! i was starting to think about doing this fix haha

kwen2501 commented:
@pytorchbot merge

kwen2501 committed:
Update on "[pipelining] Retire PIPPY_VERBOSITY in favor of TORCH_LOGS=pp"


https://github.com/pytorch/pytorch/pull/126499/ established: 

`TORCH_LOGS=pp` --> info
`TORCH_LOGS=-pp` --> warn
`TORCH_LOGS=+pp` --> debug

cc mrshenli pritamdamania87 zhaojuanmao satgera gqchen aazzolini osalpekar jiayisuse H-Huang awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k

[ghstack-poisoned]

kwen2501 commented:
@pytorchbot merge
